p4include/psa.p4

```diff
@@ -248,6 +223,36 @@ extern PacketReplicationEngine {
   */
 }
 
+extern Clone {
+  /// create a copy of the packet to the specified mirror session.
```

Comment:
Need to capture the rules on how to define headers with leading session_id/clone_id and what to do in the parser. The example you had was good to clarify.